CHANGELOG.md

```diff
@@ -62,6 +62,8 @@
   walking performance.
 - [#3178](https://github.com/ChainSafe/forest/issues/3178): Removed inaccurate
   progress log ETA; now only the elapsed time is displayed.
+- [#3341](https://github.com/ChainSafe/forest/issues/3341): Bumped positioned-io
+  and removed redundant abstractions.
```

Comment:
This is completelely internal to Forest codebase.
> - if the change does not directly affect the user, it should not be included in
  the changelog - for example, refactoring of the codebase,
```suggestion
```